packages/bsky/tests/views/profile.test.ts

```diff
@@ -225,6 +227,154 @@ describe('pds profile views', () => {
     })
   })
 
+  describe('status', () => {
+    const embed: AppBskyEmbedExternal.External = {
+      $type: 'app.bsky.embed.external#external',
```

Comment:
My understanding is that this is intended to be a `AppBskyEmbedExternal.Main` with `$type: 'app.bsky.embed.external'`.